### ISSUE TITLE
[Bug] Stretchy multi-tool (Size doesnt matter)

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Starship-EVO#2096]
### Short Description of the issue:
I was playing the game, not doing anything in particular, building, firing lasers etc. Then the multi-tool got a bit weird whenever I was switching from third-person to first-person view etc.
Eventually, the tool became so big, even when I wasn't holding anything I could still see it.

### Steps to reproduce the issue:

Not really sure at the moment... just sorta happened..

### Environment
- Starship EVO version: 20w21a
- Graphics Card: 2080 Super OC

### Additional information:

![20200518152322_1](https://user-images.githubusercontent.com/33239800/82184933-c66e9200-992b-11ea-8c7b-bc648aa1f0a5.jpg)
![20200518152404_1](https://user-images.githubusercontent.com/33239800/82184941-cb334600-992b-11ea-9550-58da5d9131b4.jpg)
![20200518152510_1](https://user-images.githubusercontent.com/33239800/82184953-cec6cd00-992b-11ea-87c7-912261c4182b.jpg)

See normal sized multitool on components shelf in background for scale..
![20200518152632_1](https://user-images.githubusercontent.com/33239800/82185029-ebfb9b80-992b-11ea-9b83-713d55b5bad4.jpg)

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4642602/output_log.txt)

**Comment by Sullos** [comment]
Update.. it appears to get smaller at times too...?
![20200518174105_1](https://user-images.githubusercontent.com/33239800/82186892-e81d4880-992e-11ea-9505-576e95ffbf62.jpg)
![20200518174117_1](https://user-images.githubusercontent.com/33239800/82186913-f0758380-992e-11ea-81c9-06a662d55f51.jpg)
![20200518174123_1](https://user-images.githubusercontent.com/33239800/82186922-f2d7dd80-992e-11ea-9142-5d906fbbed34.jpg)
though I could be wrong, and these could be normal sizes after experiencing big multi tool?

**Comment by tsunamayo** [comment]
woa this is a funky one... I will check this...

**Comment by tsunamayo** [comment]
No it is not the normal size. So do you manage to reproduce the issue several time, or was it a one off? Do you remember having changed the player race? Thanks

**Comment by tsunamayo** [comment]
@Sullos okay tell me if you see this in the latest build! Thanks

**Comment by Sullos** [comment]
Sorry Tsuna, I didn't see your last reply!
I have only been able to reproduce on two occasions. The first time I was streaming gameplay and it happened with a larger tool, then the second time was later that day about 8 hours later after I had restarted the game and it had shrunk down.
I didn't change the player race. The first time it happened I had pressed F6 and F4 a few times, though it seems to have gotten worse upon pressing F6 a few times. The second time it happened I don't recall pressing F6.
Sorry for the slow replies.
